docs/tests.md

```diff
@@ -0,0 +1,18 @@
+# Tests for actors
+
+You can implement two types of tests for an actor in the leapp-repository - functional and unit tests.
+
+## Functional tests
+   - These tests provide fabricated input messages for the actor, check the output messages and check for side effects stated in the actor's description.
+   - These tests should consider the actor to be a black box, so ideally they should written by somebody who doesn't know the code.
```

Comment:
From @AloisMahdal: do not mention black box, it may be confusing. We would need to define grey, white or black boxes.

Reply:
Reworded to get rid of "black box".